src/libraries/Common/src/Roslyn/SyntaxValueProvider_ForAttributeWithSimpleName.cs

```diff
@@ -334,7 +334,15 @@ SyntaxTreeInfo computeTreeInfo()
                         // For any other node, just keep recursing deeper to see if we can find an attribute. Note: we cannot
                         // terminate the search anywhere as attributes may be found on things like local functions, and that
                         // means having to dive deep into statements and expressions.
-                        foreach (var child in node.ChildNodesAndTokens().Reverse())
+                        var childNodesAndTokens = node.ChildNodesAndTokens();
+
+                        // Avoid performance issue (in .NET 7 and earlier) iterating the child list in reverse
```

Comment:
How does this relate to ".NET 7"?  Is this actually about the version of Roslyn that's being used?

Reply:
Yes, this comment is about the version of Roslyn that's used.

Perhaps this should be: "Avoid performance issue _in earlier implementations of ChildSyntaxList_ iterating the child list in reverse."

Reply:
I see.  You could just be explicit and include a link to the Roslyn PR that fixed it.  Essentially we're saying if you're running with an earlier Roslyn than that, you could have perf issues.  I'm just questioning the original wording because you will likely end up getting that fix as part of .NET 7 SDK updates and as part of VS updates, right?